encoding/protobq/schema.go

```diff
@@ -80,6 +83,27 @@ func (o SchemaOptions) inferFieldSchema(field protoreflect.FieldDescriptor) *big
 	return fieldSchema
 }
 
+func appendNonEmpty(parts []string, value string) []string {
+	trimmed := strings.TrimSpace(value)
+	if len(trimmed) > 0 {
+		parts = append(parts, trimmed)
+	}
+	return parts
+}
+
+func (o SchemaOptions) buildDescription(field protoreflect.FieldDescriptor) string {
+	if !o.UseProtoCommentsAsDescription {
+		return ""
+	}
+	sourceLocation := field.ParentFile().SourceLocations().ByDescriptor(field)
+	var parts []string
+	for _, comment := range strings.Split(sourceLocation.LeadingComments, "\n") {
+		parts = appendNonEmpty(parts, comment)
+	}
+	parts = appendNonEmpty(parts, sourceLocation.TrailingComments)
+	return strings.Join(parts, "\n")
```

Comment:
So, this is to get rid of empty rows in the comment? Or is it for something more/else?

Reply:
Yes, exactly.